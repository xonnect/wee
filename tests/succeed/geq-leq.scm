;; test to see if the newly reintroduced <= and >= work

(if (and (<= 45 45)
	 (<= 45 47)
	 (not (<= 45 32))
	 (>= 32 10)
	 (>= 32 32)
	 (not (>= 32 45)))
    (display "PASSED\n"))
(display (<= 45 45))
(display (<= 45 47))
(display (not (<= 45 32)))
(display (>= 32 10))
(display (>= 32 32))
(display (not (>= 32 45)))
(display "\n")
