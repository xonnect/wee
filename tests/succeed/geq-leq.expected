PASSED
#t#t#t#t#t#t
